Makefile

```diff
@@ -0,0 +1,9 @@
+init:
+    pip install pipenv
```

Comment:
This `Makefile` is a part of getting Travis CI running. Perhaps it would be best if I pushed all these requirements into the `travis.yml` file so there is less confusion about how to use it.